apps/velo-external-db/test/e2e/app_data_hooks.e2e.spec.ts

```diff
@@ -518,14 +518,13 @@ describe(`Velo External DB Data Hooks: ${currentDbImplementationName()}`, () =>
                 dataHooks: {
                     beforeAll: (_payload, _requestContext, _serviceContext) => {
                         const error = new Error('message')
-                        error['status'] = '409'
```

Comment:
I don't think it's right to set status to error ... I think an internal error (status 500) should be thrown every time the client code in hooks throws.